### DISCUSSION TITLE
vim/neovim setup

### DISCUSSION THREAD

**Discussion by Jasha10** [nushell/nushell#14183] [General]
Here are my current (neo)vim settings for nushell compatibility:
```
if &shell =~# '\<nu\>$'
  set shellredir="out>"
  set shellquote="'"
  set shellxquote=""
  set shellcmdflag=-c
endif
```
Here are my open questions:
- How should I set the vim `'shellpipe'` option?

**Comment by fdncred** [comment]
I think we have a setup in our [integrations](https://github.com/nushell/integrations/tree/main/nvim) repo and a different one in our [tree-sitter-nu](https://github.com/nushell/tree-sitter-nu/blob/main/installation/neovim.md) repo. You might check there. What you're showing looks familiar. I don't use nvim except for playing around with.
